test/integration/backward_compatible/parallel/ListenersOnReconnectTest.js

```diff
@@ -49,8 +49,7 @@ describe('ListenersOnReconnectTest', function () {
                 smartRouting: isSmart
             },
             properties: {
-                'hazelcast.client.heartbeat.interval': 1000,
-                'hazelcast.client.heartbeat.timeout': 3000
+                'hazelcast.client.heartbeat.interval': 1000
```

Comment:
I believe this change is not needed anymore, and can safely be put back

Reply:
I don't think so. I will run the test again with timeout added to see if you are right. I only changed 4.0.x and started the test here: https://github.com/srknzl/hazelcast-nodejs-client/actions/runs/1922054432

Reply:
Why is that?

Reply:
Nevermind I think I understood it. I will apply your suggestion

Reply:
We decided to remove this line since the last connection may close due to heartbeat timeout. Example: https://github.com/srknzl/hazelcast-nodejs-client/runs/5404032479?check_suite_focus=true